app/models/system_message.rb

```diff
@@ -29,9 +46,19 @@ def recipients
     users
   end
 
-  def send!
-    recipients.each do |user|
-      RadbearMailer.simple_message(user, "Important Message From #{I18n.t(:app_name)}", message).deliver_later
+  def send!(current_user)
+    if email?
+      recipients.each do |user|
+        RadbearMailer.simple_message(user, "Important Message From #{I18n.t(:app_name)}", message).deliver_later
+      end
+    else
+      SystemSMSJob.perform_later(message, recipients.map(&:id), current_user)
```

Comment:
If not already, make sure recipients have a phone_number - or mobile_phone, need to see what attribute the other features are using

Reply:
did this get done? I don't see it at first glance

Reply:
yep, performed the check in the job

Reply:
I mean, we should scope the recipients query to only include users that have the appropriate phone number present